<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>go2epa</class>
 <widget class="QDialog" name="go2epa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>748</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go2Epa</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_accept">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>668</width>
        <height>697</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Inp</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="2" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QGroupBox" name="grb_general">
               <property name="title">
                <string>GENERAL</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="0,0">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="grl_general_1" rowstretch="0" columnstretch="0"/>
                </item>
                <item row="0" column="1">
                 <layout class="QGridLayout" name="grl_general_2"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="grb_hydraulics">
               <property name="title">
                <string>HYDRAULICS</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="1">
                 <layout class="QGridLayout" name="grl_hyd_4"/>
                </item>
                <item row="0" column="0">
                 <layout class="QGridLayout" name="grl_hyd_3"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="grb_quality">
               <property name="title">
                <string>QUALITY</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="grl_quality_5"/>
                </item>
                <item row="0" column="1">
                 <layout class="QGridLayout" name="grl_quality_6" columnstretch="0"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="grb_valve_status">
               <property name="title">
                <string>VALVE STATUS</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="1">
                 <layout class="QGridLayout" name="grl_status_8"/>
                </item>
                <item row="0" column="0">
                 <layout class="QGridLayout" name="grl_status_7"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="grb_crm">
               <property name="title">
                <string>CRM</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_16">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="grl_crm_9"/>
                </item>
                <item row="0" column="1">
                 <layout class="QGridLayout" name="grl_crm_10"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="grb_din">
               <property name="title">
                <string>DINAMIC WAVE</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_11">
                <item row="1" column="0">
                 <layout class="QGridLayout" name="grp_dinw_11"/>
                </item>
                <item row="1" column="1">
                 <layout class="QGridLayout" name="grp_dinw_12"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="grb_time_steps">
               <property name="title">
                <string>DATE &amp; TIME STEPS</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_19">
                <item row="0" column="1">
                 <layout class="QGridLayout" name="grl_date_14" rowstretch="0" columnstretch="0"/>
                </item>
                <item row="0" column="0">
                 <layout class="QGridLayout" name="grl_date_13"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="grb_other">
               <property name="title">
                <string>OTHER</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_12">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="grl_inpother_15"/>
                </item>
                <item row="0" column="1">
                 <layout class="QGridLayout" name="grl_inpother_16"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Rpt</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QGroupBox" name="grb_reports">
               <property name="title">
                <string>REPORTS</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="grl_reports_17"/>
                </item>
                <item row="0" column="1">
                 <layout class="QGridLayout" name="grl_reports_18"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="grb_rpt_others">
               <property name="title">
                <string>OTHER</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_14">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="drl_rptother_19"/>
                </item>
                <item row="0" column="1">
                 <layout class="QGridLayout" name="drl_rptother_20"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
